src/autoscaler/cf/service_instances.go

```diff
@@ -4,35 +4,25 @@ import (
 	"fmt"
 )
 
-const (
-	ServicePlansPath = "v2/service_plans"
+type (
+	ServicePlanData struct {
+		Guid string `json:"guid"`
+	}
+	ServicePlan struct {
+		Data ServicePlanData `json:"data"`
+	}
+	ServiceInstanceRelationships struct {
+		ServicePlan ServicePlan `json:"service_plan"`
+	}
+	ServiceInstance struct {
+		Guid          string                       `json:"guid"`
+		Type          string                       `json:"type"`
+		Relationships ServiceInstanceRelationships `json:"relationships"`
+	}
 )
 
-type ServiceInstanceEntity struct {
-	ServicePlanGuid string `json:"service_plan_guid"`
-}
-
-type ServiceInstanceResource struct {
-	Entity ServiceInstanceEntity `json:"entity"`
-}
-type ServicePlanData struct {
-	Guid string `json:"guid"`
-}
-type ServicePlan struct {
-	Data ServicePlanData `json:"data"`
-}
-type ServiceInstanceRelationships struct {
-	ServicePlan ServicePlan `json:"service_plan"`
-}
-type ServiceInstance struct {
-	Guid          string                       `json:"guid"`
-	Type          string                       `json:"type"`
-	Relationships ServiceInstanceRelationships `json:"relationships"`
-}
-
 func (c *Client) GetServiceInstance(serviceInstanceGuid string) (*ServiceInstance, error) {
-
-	theUrl := fmt.Sprintf("/v3/service_instances/%s?fields[service_plan]=name,guid", serviceInstanceGuid)
+	theUrl := fmt.Sprintf("/v3/service_instances/%s", serviceInstanceGuid)
```

Comment:
fields[service_plan] is actually useless since the plan guid is alreay in the response it just adds the name which is useless to us